azext_iot/_params.py

```diff
@@ -206,6 +206,22 @@ def load_arguments(self, _):
         context.argument('content_type', options_list=['--content-type', '--ct'],
                          help='MIME Type of file.')
 
+    with self.argument_context('iot device get-parent') as context:
+        context.argument('device_id', help='This is an non-Edge device.')
```

Comment:
Rephrase it to 'Id of non-edge device'. I also notice sometimes we use Edge and sometimes edge. Can you find out what we currently use in existing commands and be consistent here?

Reply:
Fixed